pallets/order-book/src/lib.rs

```diff
@@ -369,24 +401,40 @@ pub mod pallet {
         #[pallet::weight(<T as Config>::WeightInfo::place_limit_order())]
         pub fn place_limit_order(
             origin: OriginFor<T>,
-            order_book_id: OrderBookId<T>,
-            _price: OrderPrice,
-            _amount: OrderVolume,
-            _side: PriceVariant,
-            _lifespan: T::Moment,
+            order_book_id: OrderBookId<AssetIdOf<T>>,
+            price: OrderPrice,
+            amount: OrderVolume,
+            side: PriceVariant,
+            lifespan: T::Moment,
         ) -> DispatchResult {
-            let _who = ensure_signed(origin)?;
-            let _order_book =
+            let who = ensure_signed(origin)?;
+            let mut order_book =
                 <OrderBooks<T>>::get(order_book_id).ok_or(Error::<T>::UnknownOrderBook)?;
-            // todo (m.tagirov)
-            todo!()
+            let dex_id = order_book.dex_id;
+            let order_id = order_book.next_order_id();
+            let now = pallet_timestamp::Pallet::<T>::now();
```

Comment:
I think you should use UnixTime trait in Config instead of direct access to timestamp pallet

Reply:
I already tried